package.json

```diff
@@ -158,8 +158,8 @@
     "@microsoft/api-extractor": "^7.52.2",
     "@peculiar/webcrypto": "^1.5.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
-    "@rspack/cli": "^1.3.0",
-    "@rspack/core": "^1.3.0",
+    "@rspack/cli": "^1.3.6",
```

Comment:
the libs that are upgraded contain `path-to-regexp` versions that are vulnerable.